src/main/java/ladder/domain/Ladder.java

```diff
@@ -5,20 +5,20 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-public class Radder {
+public class Ladder {
     public static final int MIN_HEIGHT = 1;
     private List<Line> lines;
 
-    public Radder(List<Line> lines) {
+    public Ladder(List<Line> lines) {
```

Comment:
```suggestion
    private Ladder(List<Line> lines) {
```

정적 팩터리 메서드를 제공할 경우 private으로 구현해도 되지 않을까?

Reply:
사실 테스트 케이스를 위해 public 선언해 둔 건데요. (임의가 아닌 지정된 사다리값을 입력받기 위해)
이런 경우는 생성자를 private 두고, 지정된 값으로 사다리를 생성하는 정적 팩토리 메소드를 추가하는 방향이 좋은 걸까요?